common/reviews/api/gulp-core-build-typescript.api.ts

```diff
@@ -22,29 +22,27 @@ interface ITsConfigFile<T> {
 
 // @public
 class TypeScriptConfiguration {
-  public static fixupSettings(compilerOptions: ts.Settings,
```

Comment:
>ublic static  [](start = 3, length = 13)

why is this not public static anymore

Reply:
After this change, API Extractor processes the *.d.ts files instead of *.ts files.  The compiler strips the "public" keyword from the *.d.ts files.  I was going to make a change to bring it back, but then it occurred to me that maybe the `public` keyword isn't that useful in this context:  Since `private` is stripped from the reports, `protected` and `public` are the only possibilities, and `public` looks a little strange in this situation:

```typescript
   /** @internal */
  public doSomething();  // is it public? or is it internal?
```